### ISSUE TITLE
Problem training for new detection task

### ISSUE THREAD

**Issue by mtheresepersson** [experiencor/keras-yolo2#297]
Hi,
First of all thanks for the great work! I'm trying to train the network to detect white blood cells instead of red blood cells but I'm having some problems getting it to work. Right now I'm trying to overfit to 5 images just to make sure that it can detect white blood cells at all, but it just predicts zero boxes or way too many boxes in the wrong part of the image. All images I'm training with contains one white blood cell (it's the 5 first images from this blood cell dataset). 

This method seems to work fine for detecting red blood cells, this is the result I get when running one warm-up epoch, 15 epochs and batch-size 1. 

![bloodimage_00000_detected_rbc_15_epochs](https://user-images.githubusercontent.com/33990964/40445956-76bd7ac2-5e82-11e8-964a-d77ceaff61ad.jpg)

However, when I run with exactly the same images and settings but with labels for white blood cells it doesn't detect any boxes but the error goes down to 0.00769. I'm using the same images for training and validation (since I want to overfit). I'm predicting on the first image in the training set. 

My labels for white blood cells are in the same format as the labels for red blood cells. Here's an example of how my true boxes look like

![true_neutrophil](https://user-images.githubusercontent.com/33990964/40446157-270396c8-5e83-11e8-8641-66790c006a49.png)

I don't know what I'm doing wrong, do you have any ideas? All help would be appreciated!
